charts/brigade/values.yaml

```diff
@@ -73,7 +73,7 @@ worker:
 gw:
```

Comment:
I'd rather want to refrain from a ad-hoc rename of `gw` to e.g. `github` here, I'll probably address it in an another PR once we get some consensus on it.

Would it be nice to reorganize this and `cr` altogether under a new `gateway` key like:

```yaml
gateway:
  github:
    enabled: true
    registry: deis
    name: brigade-github-gateway
    #tag
    #pullPolicy: IfNotpresent
    #buildForkedPullRequests: true
  cr:
    enabled: false
     # ...
```